src/traces/parcoords/parcoords.js

```diff
@@ -227,8 +234,6 @@ function styleExtentTexts(selection) {
     selection
         .classed('axisExtentText', true)
         .attr('text-anchor', 'middle')
-        .style('font-weight', 100)
```

Comment:
We don't set `font-weight` anyway in our code - though inputting text as `<b>text</b>` effectively does [that](https://github.com/plotly/plotly.js/blob/ce970f112a8fff0d09909529be8a713f5d4f36cc/src/lib/svg_text_utils.js#L229). 

This is another thing I wished I would've noticed while reviewing your first parcoords PR. Oh well. I think it's best to drop `font-weight` and make parcoords look a little more plotly-esque.